pkg/credsgen/in_memory_generator/certificate_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("InMemoryGenerator", func() {
 			)
 
 			BeforeEach(func() {
-				ca, _ = generator.GenerateCertificate("testca", credsgen.CertificateGenerationRequest{IsCA: true})
+				ca, _ = generator.GenerateCertificate("testca", credsgen.CertificateGenerationRequest{CommonName: "Fake CA",IsCA: true})
```

Comment:
Could we add a test that asserts that the given `CommonName` is actually used for CAs? I think this could go to the test in line 122.

Reply:
Done. Add asserts about CN.